src/interfaces/IEarningPowerCalculator.sol

```diff
@@ -1,14 +1,44 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.23;
 
-/// @notice Interface for calculating earning power of a staker based on their delegate score in
-/// Staker.
+/// @title IEarningPowerCalculator
+/// @author [ScopeLift](https://scopelift.co)
+/// @notice Interface to which Earning Power Calculators must conform in order to provide earning
+/// power updates to an instance of Staker. Well behaving earning power calculators should:
+/// 1. Be deterministic, i.e. produce the same output for the same input at a given time.
+/// 2. Return values that are in the same order of magnitude as the amount staked, i.e. avoid
+///    returning values that are dramatically detached from the staked amount.
```

Comment:
It would be good if we were a little more specific what dramatically different means here. Does it mean 200%+ of the stake amount? Returning 0 for a large stake could be interpreted as an order of magnitude different and dramatically detached from the stake amount, but we wouldn't consider the calculator to be misbehaving.